cookies/giles.py

```diff
@@ -654,10 +687,36 @@ def process_details(data, upload_id, username):
                                      content_type=fmt_data.get('content-type'),
                                      name='%s (%s)' % (page_resource.name, fmt))
 
+        # Content resource for each additional file, if available.
+        for additional_file in page_data.get('additionalFiles', []):
+            content_type = additional_file.get('content-type')
+            uri = '%s/files/%s' % (giles, additional_file.get('id'))
```

Comment:
is there a reason you build the url differently here than you do above?

Reply:
I followed the existing logic for URLs. I will unify it.